### ISSUE TITLE
[Docs]: add TLDR for commit Message Format in contributing.md

### ISSUE THREAD

**Issue by calvinalkan** [snicco/snicco#145]
### Thanks for taking the time to fill out this documentation request!"

- [X] I have searched the issue tracker for similar issues
- [X] I have searched the documentation of the relevant package(s)

### Which @snicco/* package(s) have missing/wrong/unclear documenation?

monorepo

### Which part of the documentation is unclear, missing or wrong?

Add a TL;DR to the commit message section.

**Comment by szepeviktor** [comment]
@calvinalkan Still no TL;DR section in https://github.com/snicco/snicco/blob/master/CONTRIBUTING.md#commit-message-format

**Comment by calvinalkan** [comment]
@szepeviktor Hey, 

TBH, I am curious how we could further trim this down. 

https://github.com/snicco/snicco/blob/master/CONTRIBUTING.md#examples

We even integrated the [commitizen](https://github.com/commitizen/cz-cli) cli that does all of this interactively. 

You can use `npm run commit` or `npx cz `to prompt the cli.

At this point, relaxing the commit message format is not an option. The entire CI and release pipeline depends on this. 

But feel free to open a PR if you feel this could be made easier for new people.

**Comment by szepeviktor** [comment]
Just add
```markdown
### TL;DR

Example commit message ...
```
